include/psp2/usbd.h

```diff
@@ -7,89 +7,505 @@
 #ifndef _PSP2_USBD_H_
 #define _PSP2_USBD_H_
 
-#include <psp2/kernel/threadmgr.h>
-
 #ifdef __cplusplus
 extern "C" {
 #endif
 
 typedef enum SceUsbdErrorCode {
-	SCE_USBD_ERROR_IO            = 0x80240001,
-	SCE_USBD_ERROR_INVALID_ARG   = 0x80240002,
-	SCE_USBD_ERROR_ACCESS        = 0x80240003,
-	SCE_USBD_ERROR_NO_DEVICE     = 0x80240004,
-	SCE_USBD_ERROR_NOT_FOUND     = 0x80240005,
-	SCE_USBD_ERROR_BUSY          = 0x80240006,
-	SCE_USBD_ERROR_TIMEOUT       = 0x80240007,
-	SCE_USBD_ERROR_OVERFLOW      = 0x80240008,
-	SCE_USBD_ERROR_PIPE          = 0x80240009,
-	SCE_USBD_ERROR_INTERRUPTED   = 0x8024000A,
-	SCE_USBD_ERROR_NO_MEM        = 0x8024000B,
-	SCE_USBD_ERROR_NOT_SUPPORTED = 0x8024000C,
-	SCE_USBD_ERROR_FATAL         = 0x802400FF
+    SCE_USBD_ERROR_NOT_INITIALIZED     = 0x80240001,
+    SCE_USBD_ERROR_ALREADY_INITIALIZED = 0x80240002,
+    SCE_USBD_ERROR_INVALID_PARAM       = 0x80240003,
+
+    SCE_USBD_ERROR_PIPE_NOT_FOUND      = 0x80240004,
+
+    SCE_USBD_ERROR_NO_MEMORY           = 0x80240005,
+    SCE_USBD_ERROR_DEVICE_NOT_FOUND    = 0x80240006,
+
+    SCE_USBD_ERROR_80240007            = 0x80240007, //
+    SCE_USBD_ERROR_80240009            = 0x80240009, //
+    SCE_USBD_ERROR_8024000A            = 0x8024000A, //
+
+    SCE_USBD_ERROR_FATAL               = 0x802400FF
```

Comment:
are old names wrong?
and also maybe we need to define `deprecated` marking

Reply:
Old names are copypasted from ps4 and are totally wrong